packages/apps/plugins/plugin-deck/src/DeckPlugin.tsx

```diff
@@ -45,19 +46,24 @@ const isSocket = !!(globalThis as any).__args;
 // TODO(mjamesderocher): Can we get this directly from Socket?
 const appScheme = 'composer://';
 
-// TODO(burdon): Evolve into customizable prefs, but pls leave for demo.
-const customSlots: DeckLayoutProps['slots'] = {
+const getSlots = ({ themeBackgroundDark, themeBackgroundLight }: DeckSettingsProps): DeckLayoutSlots => ({
   wallpaper: {
-    classNames:
-      'bg-cover bg-no-repeat dark:bg-[url(https://cdn.midjourney.com/3865ba61-f98a-4d94-b91a-1763ead01f4f/0_0.jpeg)]',
+    backgroundImage: {
+      dark: themeBackgroundDark,
+      light: themeBackgroundLight,
+    },
   },
   deck: {
-    classNames: 'px-96 bg-neutral-50 __dark:bg-neutral-950 dark:bg-transparent dark:opacity-95',
+    classNames: mx(
+      'px-96',
+      // TODO(burdon): Option for opacity.
+      themeBackgroundDark.length ? '_dark:opacity-95 _dark:bg-transparent' : 'bg-neutral-50 dark:bg-neutral-950',
+    ),
   },
   plank: {
     classNames: 'mx-1 bg-neutral-25 dark:bg-neutral-900',
   },
-};
+});
```

Comment:
This implementation should be improved now that this isn’t for a demo — I think there are a variety of options we could go with, depending on what the feature is meant to work / how it is meant to scale. I think we should discuss.

Reply:
Yep let's discuss and do this properly.